.github/workflows/dependency-review.yml

```
@@ -0,0 +1,11 @@
+name: 'Dependency Review'
+on: [pull_request]
+
+jobs:
```

It would be nice to add a permissions block here:

```suggestion
permissions:
  actions: read
  pull-requests: read

jobs:
```

I'm not sure if these are the correct permissions.